test_task_packed_picking/readmy/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,2 @@
+* Raphaël Reverdy <sebastien.beau@akretion.com>
```

Comment:
No need for this file. Especially taking into account that you are not Raphaël Reverdy )